documentation/project-docs/external-component-telemetry.md

```diff
@@ -0,0 +1,39 @@
+# Responsibly managing telemetry in external components
+
+Many components are _delivered_ with or by the .NET SDK but want to collect and
+manage telemetry. The SDK has telemetry collection mechanisms that may appear
+attractive, but present down-sides for these authors.
```

Comment:
nit: use of multiple lines that are not rendered in Markdown.

Reply:
I've picked up this habit of trying to split early because it makes diffs and single-line changes more readable for folks editing in GitHub